src/main/kotlin/de/itemis/mps/gradle/generate/Plugin.kt

```diff
@@ -79,8 +85,8 @@ open class GenerateMpsProjectPlugin : Plugin<Project> {
                     // git4idea: has to be on classpath as bundled plugin to be loaded (since 2019.3)
                     classpath(fileTree(File(mpsLocation, "/plugins")).include("git4idea/**/*.jar"))
                     classpath(genConfig)
-                    debug = extension.debug
-                    main = "de.itemis.mps.gradle.generate.MainKt"
+                    debug = extension.debug.getOrElse(false)
```

Comment:
Should probably use convention for `extension.debug` as well?

Reply:
done